### PR TITLE
Pagination styling and a11y-friendly hiding/showing

### PR DESCRIPTION
**Pull request by mperrotti**
@akdetrick - Please take a good look at what's going on with "visualHide" and "visualShow" in:
sass/util/_display.scss
sass/modifiers/_media-conditional.scss
sass/modifiers/_display.scss

Does what I added make sense to have in SQ2? If not, let's try and figure out something better to do for hiding and showing page numbers at various sizes

### CODE REVIEW

File: sass/modifiers/_display.scss [hunk 1]
```diff
@@ -12,7 +12,8 @@ Class                   | Description
 `.display--block`       | sets `display` to `block`
 `.display--inline`      | sets `display` to `inline`
 `.display--inlineBlock` | sets `display` to `inlineblock`
-
+`.display--visualHide`  | visually hides an element
```
Comment (akdetrick) [inline sass/modifiers/_display.scss]: i'd just change the documentation here to explain that the element won't be in document flow, making it more similar to `display: none` than `visibility: hidden`

File: sass/modifiers/_display.scss [hunk 2]
```diff
@@ -20,3 +21,24 @@ Class                   | Description
...
 }
+
+@include _modifier(display, visualHide) {
+	border: 0;
+	clip: rect(0 0 0 0);
+	position: absolute;
+	overflow: hidden;
+	margin: -1px;
+	padding: 0;
+	width: 1px;
```
Comment (akdetrick) [inline sass/modifiers/_display.scss]: will this work with a height and width of `0px`?

File: sass/modifiers/_display.scss (hunk 1 above)
Comment (akdetrick) [inline sass/modifiers/_display.scss]: any objection to calling this `visibility--a11yHidden`/`visibility--a11yHidden` and moving it to a new `_visibility.scss` partial?

It doesn't modify the `display` property as the other display classes do; it does some extra neat stuff.

**Comment by mperrotti** [comment]
I think that makes sense - no objection here.

**Pull request merged**